src/utils/query/Query.tsx

```diff
@@ -37,5 +37,11 @@ export const useQuery = <T,>(
     dispatch({ tag: "fetch" });
   }, [dispatch]);
 
+  document.addEventListener("visibilitychange", () => {
+    if (document.visibilityState === "visible") {
+      refetch();
+    }
+  });
```

Comment:
Object `document` ini gak ada di server. Ini bakal break kalau pake SSR. Perlu dibungkus useEffect

Reply:
Ahh I see, I missed that one 🏀